### PR TITLE
WIP: Update docs.md

### PR DESCRIPTION
**Pull request by hachi8833**
This PR is a continuation of #3637, including additional descriptions.

I'd be glad if some of committers would check this, especially around `module`

Ref: 

* https://github.com/vlang/v/issues/2685#issuecomment-551939613
* https://github.com/vlang/v/issues/2685#issuecomment-551940824

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -847,13 +948,78 @@ which module is being called. Especially in large code bases.
...
+
+You can rename the modules when imported:
+
+```v
+import module_name as mn      // mn is an alias of the module_name
+```
+
+Note that V looks up the module by the containing directory name, which means the directory name should match exactly the module name.
+
+The name of the modules under the directory can be any `*.v`, but at least putting the file with the exact module name is preferable as a convention because the module file with the exact name, where `init()` function (see below) can exist, will be fetched first.
```
Comment (spytheman) [inline doc/docs.md]: > where `init()` function (see below) can exist, will be fetched first.
That is not so.
The filenames in a module just need to end with `.v` . The init() function for a module can be anywhere inside one of the `.v` files.

File: doc/docs.md [hunk 2]
```diff
@@ -847,13 +948,78 @@ which module is being called. Especially in large code bases.
...
+
+TBD: You can create modules anywhere.
+
+The lookup for modules happens in this order:
+
+1. project's local folders (like `./my_module/`)
+2. v's `vlib/` folder (like `vlib/my_module/`)
+3. home directory's `~/.vmodules/` folder (like `~/.vmodules/my_module/`)
+
+You can specify modules in subdirectory with `.` (not with `/`) like`module_name.submodule_name`:
```
Comment (spytheman) [inline doc/docs.md]: > `like`module_name.submodule_name`

I think that better be:
> like`import module_name.submodule_name`

File: doc/docs.md (hunk 1 above)
Comment (hachi8833) [reply doc/docs.md]: Thank you! Fixed as suggested.

File: doc/docs.md (hunk 2 above)
Comment (hachi8833) [reply doc/docs.md]: Fixed as suggested.

**Comment by hachi8833** [comment]
Looks the PR's been obsolete. Closing...

**Pull request closed without merging**